src/main/java/edu/harvard/iq/dataverse/usage/StringUtil.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cn.edu.pku.lib.util;
```

Comment:
I'm seeing "[ERROR] Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.0.0:report (default-cli) on project dataverse: I/O operation failed: No source found for cn/edu/pku/lib/util/StringUtil.java -> [Help 1]" at https://travis-ci.org/IQSS/dataverse/builds/96229791#L3026

@bencomp and I are surprised that the Travis build passed: http://irclog.iq.harvard.edu/dataverse/2015-12-11